FlowCrypt/Functionality/WKDURLs/PubLookup.swift

```diff
@@ -67,9 +67,9 @@ class PubLookup: PubLookupType {
             email: email,
             name: keyDetail.users.first ?? email,
             pubKey: keyDetail.public,
-            pubKeyLastSig: nil, // TODO: - will be provided later
+            pubKeyLastSig: keyDetail.lastModified.map { Date(timeIntervalSince1970: Double($0)) },
             pubkeyLastChecked: Date(),
-            pubkeyExpiresOn: nil, // TODO: - will be provided later
+            pubkeyExpiresOn: keyDetail.expiration.map { Date(timeIntervalSince1970: Double($0)) },
```

Comment:
I know this is probably my problem not coming from a Swift background, but to me a `.map` implies that `keyDetail.expiration` is some sort of an iterable, and you're applying a method over it to return something iterable again.

But that's not the case here - `keyDetail.expiration` is an `Int?` and so the `.map` on values like this will never stop confusing me :-( seems to be a common practice in Swift.

Is there a pattern possible in Swift that's a bit more obvious for a non-Swifter? A ternary operator maybe? Like `keyDetail.expiration != nil ? Date(timeIntervalSince1970: Double(keyDetail.expiration)) : nil` ?

Reply:
Ok. I with there was some more obvious way (that didn't involve force unwrapping). Thanks for explanation.